metricbeat/metricbeat.reference.yml

```diff
@@ -621,7 +621,7 @@ metricbeat.modules:
 - module: mysql
   metricsets:
     - "status"
-  #  - "galera_status"
+    - "galera_status"
```

Comment:
this normally means that the metricset is disabled by default, do we really need to change this?

Reply:
I used this way (enabling metricset) to ensure that the configuration option is used with the metricset.
In this case, if I enable "galera_status", the script will include all configuration options related to it.